### ISSUE TITLE
Crayon settings save button

### ISSUE THREAD

**Issue by worldsdream** [urvanov-ru/crayon-syntax-highlighter#15]
Go to the crayon settings page. We can adjust a lot of settings and we can change the design. In the top of the crayon settings page, we can see how it will look. 

Issue: Change the theme or settings, it’s updating in the top of the crayon settings page +  it’s updating on the website. This is not great. 

What it should look like:
Change the theme or settings, it’s updating in the top of the crayon settings page. That’s it.

Now when we click the save button in crayon settings, it needs to apply to the website. Then everyone is able to see it. 

Can you reproduce this and what do you think?

**Comment by urvanov-ru** [comment]
Yes, I see page refresh. It was original Crayon behaviour. Maybe I will fix it, but that needs a lot of time. It will be not now, and it will be not fast.
